src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -53,8 +53,8 @@ public static Person[] getSamplePersons() {
 
     public static Project[] getSampleProjects() {
         return new Project[] {
-            new Project(new ProjectName("Project Alpha"), new ProjectId("1")),
-            new Project(new ProjectName("Project Beta"), new ProjectId("2")),
+            new Project(new ProjectName("Project Alpha"), new ProjectId("1"), getSkillSet("Frontend")),
+            new Project(new ProjectName("Project Beta"), new ProjectId("2"), null),
```

Comment:
Will this throw a NullPointerException, since the constructor has the line `this.skills.addAll(skills);`?

Reply:
Perhaps it will be more appropriate to pass in an empty skillSet?